testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1448,3 +1448,26 @@ def server_secret
   command = "sed -i '$d' /etc/hosts && sed -i '$d' /etc/hosts"
   $server.run(command)
 end
+
+# select an item from any dropdown
+And(/^I select "(.*?)" from "([^"]*)" dropdown/) do |selection, label|
+  # let the the select2js box filter open the hidden options
+  xpath_query = "//select[@name='#{label}']"
+  raise "xpath: #{xpath_query} not found" unless find(:xpath, xpath_query).click
+  # select the desired option
+  raise "#{label} #{selection} not found" unless find(:xpath, "//select[@name='#{label}']/option[contains(text(), '#{selection}')]").click
+end
+
+# check if a certain link is present in the channel mappings table
+Given(/^I see a "(.*?)" link in the table/) do |link_name|
+  page.has_field?("#{link_name}", with: 'Operating System')
+end
+
+Then(/^I should see a "(.*?)" link in the table/) do |link_name|
```

Comment:
Do we need 2 of the exact same step? Honest question, even if they use different starters